Cod3rsGrowth.Testes/SingletonMarca.cs

```diff
@@ -49,7 +51,7 @@ public static List<Marca> Instancia
         {
             get 
             {
-                if (InstanciaMarca == null)
+                if (Instancia.Any())
```

Comment:
Já sei que esse o teste de Marca não deve estar certo por isso aqui

Reply:
não entendi, deveria ser null? se for null continua gerando retorno nulo